introduction.md

```diff
@@ -1,3 +1,7 @@
 Bow to your master!
 
-Favourite path at Code School is Ruby of course.
\ No newline at end of file
+Favourite path at Code School is Ruby of course.
+
+*Git
+*JavaScript
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?